compiler/rustc_error_codes/src/error_codes.rs

```diff
@@ -513,7 +513,9 @@ E0790: include_str!("./error_codes/E0790.md"),
 E0791: include_str!("./error_codes/E0791.md"),
 E0792: include_str!("./error_codes/E0792.md"),
 E0793: include_str!("./error_codes/E0793.md"),
-;
+}
+
+// Undocumented removed error codes. Note that many removed error codes are documented.
```

Comment:
How can a removed error code be documented? I'm trying to remove an error code in https://github.com/rust-lang/rust/pull/115277 and tidy complains that the error code exists but isn't used. The only way I can see to fix that is to remove the documentation as well.

Reply:
https://github.com/rust-lang/rust/pull/115278 should help

Reply:
This list is used for `--explain`, so even if not emitted by rustc, it's normal to have them. So you never remove an entry unless it doesn't have an error code explanation.